www/layouts/partials/ocw_news.html

```diff
@@ -12,7 +12,7 @@
         <div class="{{ $breakpointVisibilityClass }}">
           <div id="{{ $carouselId }}" class="carousel slide">
             <div class="carousel-header d-flex flex-row justify-content-between font-weight-bold text-uppercase">
-              <h2 class="branded"><span>OCW</span> News</h2>
+              <h2 class="branded">OCW News</h2>
```

Comment:
@pdpinch `OCW News` and `OpenCourseWare Testimonies title` doesn't seems to be consistent with `NEW COURSES` title, as they have this "branded" class which reduces their font-weight.

Should we make them consistent with `NEW COURSES` title by removing this "branded" class or vice versa?

Reply:
Sure